CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+- `tools/`: Internal developer tools, such as the Monoweb Database Migrator
+
+We use [PNPM workspaces](https://pnpm.io/workspaces) along with [Turborepo](https://turbo.build/repo/docs) to manage the
+monorepo. All dependencies for all libraries and applications are managed by PNPM.
+
+All of the code in the Monorepo is TypeScript.
+
+<details>
+<summary>How are the libraries packaged and consumed?</summary>
+
+You might wonder how we build the libraries so that the applications can consume them. The answer is that we don't. We
+export the libraries as TypeScript source code, and consume them as TypeScript source code. In the default TypeScript
+compiler, this would pose a problem, since the compiler does not want to enter `node_modules` directories.
+
+However, since we use the Next.js Compiler for the web applications, and TSup for the other applications, we can tell
+the Next.js Compiler or TSup compiler to bundle the libraries as part of the build process.
+
+In Next.js, this is done by adding `transpilePackages` in the `next.config.mjs` files. In Tsup builds, we simply tell it
+to bundle everything into a single file.
+
+Examples of both can be found in the different applications in the `apps/` directory. For example, `apps/web` uses
+Next.js, and `apps/gateway-email` uses TSup.
+</details>
+
+## Tools
+
+The following tools are used to develop Monoweb:
+
+- [Node.js](https://nodejs.org/): JavaScript runtime
+- [PNPM](https://pnpm.io/): Package manager (`npm i -g pnpm`)
+- [Docker](https://www.docker.com/): Containerization for a few applications (not required `apps/web`)
+- [Docker Compose](https://docs.docker.com/compose/): For using local PostgreSQL if wanted (modern versions of Docker
+  include Docker Compose)
+- [Doppler](https://doppler.com/): Secrets management, used for retrieving secrets in development.
+    - If you are not a Dotkom team member, you will not have access to Doppler. You can still run the applications, but
+      you will have to provide the secrets yourself. In this case, you can use a `.env` file in the different
+      application directories, or export them into your shell.
+- [AWS CLIv2](https://aws.amazon.com/cli/): Used for deploying to applications to AWS. You will need to have the correct
+  AWS credentials set up to deploy to AWS. If you are not a dotkom team member, we can unfortunately not provide you
+  with credentials.
+- [Terraform](https://www.terraform.io/): Infrastructure as code, used for managing the infrastructure. Note that the
+  Terraform configuration is located in a separate, private GitHub repository. If you are not a dotkom team member, we
+  can unfortunately not provide you with access to the repository. This is done to prevent any confidential information
+  from being leaked through [Atlantis](https://www.runatlantis.io/).
+
+## Local Development
+
+To get started with local development, ensure you have the applicable tools installed. To build and run all the
```

Comment:
```suggestion
To get started with local development, ensure you have the [applicable tools](#tools) installed. To build and run all the
```

Reply:
Maybe we can outline which tools are necessary? Like in "Running with your own PostgreSQL database"?